pages/contact.html

```diff
@@ -82,43 +82,40 @@ <h3 class="text-title m text-light">Tu Blog de Noticias</h3>
 
     <main>
         
-          <div class="container">
-              <div class="row my-2">
-                <div class="col">
-                  <form action="submit">
-                    <ul>
-                      <h3 class="">Por consultas, duda o sugerencias, comunicate con nuestros expertos.</h3>
-                    
-                        <label for="correo">Correo</label><br>
-                        <input type="text" name="correo" placeholder="example@example.com"><br>
-                        <label for="correo">Asunto</label><br>
-                        <input type="text"><br>
-                        <br>
-                        <textarea name="message" id="text-area" cols="40" rows="6" maxlength="250"></textarea><br>
-                        <br>
-                        <a class="btn btn-primary" role="button" href="./error.html">Enviar</a>    
-                      </ul>
-                    
+      <div class="container flex-column">
+        <div class="row my-2">
+          <div class="col">
+            <form class="mt-5" action="submit">
+              <ul>
+                <h3>Por consultas, duda o sugerencias, comunicate con nuestros expertos.</h3>
+                <div class="mb-2">
+                  <label for="correo">Correo</label>
+                  <input type="email" name="correo" id="correo" placeholder="example@example.com" required>
                 </div>
-                    
-                <div class="col px-4">
-                    
-                    
-                    </form>   
-                  </ul>      
-                  <div class="col col-sm-12 col-md-6 col-lg-6">
-                    <p class="text-center fw-bold text-dark">Encontranos en:</p>
-                    <p class="text-center fw-bold text-dark">West Bay Road Grand Cayman, KY1-1204, Islas Caimán.</p>
-                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2662.0291844752273!2d-81.38036064169331!3d19.336920452200122!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8f2587a1c1cc9e2f%3A0x6155f3531cdfca5a!2sYoshi%20Sushi!5e0!3m2!1ses-419!2sar!4v1665687583756!5m2!1ses-419!2sar" 
-                    width="575" height="450" style="border:0;" allowfullscreen="" loading="lazy" 
-                    referrerpolicy="no-referrer-when-downgrade"></iframe>
-                  </div>
-              </div>    
-          </div>
-  
-          <div class="espacio">
-  
+                <div class="mb-2">
+                  <label for="correo">Asunto</label>
+                  <input type="text" name="texto" id="correo" required>
```

Comment:
Id and for, are the same at the email input. Please, change it.
Can use 'asunto' as id value.